backend/app/routes/upload_config_router.py

```diff
@@ -2,88 +2,91 @@
 
 from bson.errors import InvalidId
 from fastapi import APIRouter, File, UploadFile, status, HTTPException, Depends
-from fastapi.responses import FileResponse, JSONResponse
+from fastapi.responses import JSONResponse, Response
 from bson.objectid import ObjectId
 
-from app.resources.FileService import FileService, FileType
-from app.decryptor.decryptor_service import DecryptorService, PktDecryptor
+from decryptor.file_service import FileService
```

Comment:
Chyba brakuje ścieżki względnej lub bezwzględnej. W pychamie swieci sie na czerwono
Osobiscie zamienilbym na 
`from app.decryptor.file_service import FileService` lub `from ..decryptor.file_service import FileService`

Reply:
co ciekawe u mnie się nie dośćże nie świeci to działa zarówno bez app jak i z nim

Reply:
możesz też dawać komentarz i approve jak to taka błahostka jest to chyba nie potrzbujemy nowego review